ruspy/test/test_estimation.py

```diff
@@ -56,6 +56,14 @@ def test_create_trans_mat(inputs, outputs):
     )
 
 
+def test_trans_mat_rows_one(inputs):
```

Comment:
Why all the hard-coded tests? Not just here but at several places.

```
    for _ in range(1000):

        num_states = np.random.randint(10, 1000)
        num_bins = np.random.randint(1, 5)

        p = np.random.uniform(size=num_bins)
        p = p / p.sum()

        trans_mat = create_transition_matrix(num_states, p)
        np.testing.assert_almost_equal(trans_mat.sum(axis=1), 1)
```
Even better when integrating a flexible `random_init()` function for the random request to `create_transition_matrix()`.

Reply:
Used random init function.